packages/accordions/src/elements/stepper/Stepper.tsx

```diff
@@ -5,33 +5,48 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { useRef, useEffect, forwardRef, useMemo } from 'react';
+import React, { Children, forwardRef, isValidElement, useMemo } from 'react';
 import { IStepperProps } from '../../types';
 import { StyledStepper } from '../../styled';
-import { StepperContext } from '../../utils';
+import { StepContext, StepperContext } from '../../utils';
 import { Step } from './components/Step';
 import { Label } from './components/Label';
 import { Content } from './components/Content';
 
 const StepperComponent = forwardRef<HTMLOListElement, IStepperProps>(
-  ({ isHorizontal, activeIndex, ...props }, ref) => {
-    const currentIndexRef = useRef(0);
+  ({ isHorizontal, activeIndex, children, ...props }, ref) => {
     const stepperContext = useMemo(
       () => ({
         isHorizontal: isHorizontal || false,
-        activeIndex: activeIndex!,
-        currentIndexRef
+        activeIndex: activeIndex!
```

Comment:
```suggestion
        activeIndex
```

...with `DEFAULT_ACTIVE_INDEX` set above.